contents/blog/posthog-vs-ga4.md

```diff
@@ -0,0 +1,536 @@
+        </td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>UTM tracking</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Track marketing campaigns with UTM tags</p>
+        </td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Funnels</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Visualize conversion rates and drop-offs</p>
+        </td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>User paths</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Track user flows and where they drop-off</p>
+        </td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Retention</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Analyze user and revenue retention</p>
+        </td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+PostHog also offers the following features to help you understand how people are using your product:
+
+<div className="overflow-x-auto -mx-5 px-5 w-screen md:w-auto">
+  <table className="w-full mt-4">
+    <thead>
+      <tr>
+        <td className="w-3/12"></td>
+        <td class="text-center"><strong>GA4</strong></td>
+        <td class="text-center"><strong>PostHog</strong></td>
+        <td></td>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Correlation analysis</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Automatically highlight factors affecting funnel conversion rates</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Lifecycle insights</strong></p>
```

Comment:
They do have new and returning, but maybe this is being a bit generous to them:

https://support.google.com/analytics/answer/11004084?sjid=10429453555889214577-NA#zippy=%2Cnew-users-and-returning-users

Reply:
Nice. Thanks for this, didnt realize they had it